packages/peregrine/lib/talons/CartPage/GiftCards/useGiftCard.js

```diff
@@ -0,0 +1,13 @@
+import { useCallback } from 'react';
+
+export const useGiftCard = props => {
+    const { code, handleRemoveCard } = props;
+
+    const handleRemoveCardWithCode = useCallback(() => {
+        handleRemoveCard(code);
+    }, [code, handleRemoveCard]);
```

Comment:
Am I overcomplicating things here? The `handleRemoveCard` callback function defined in the `useGiftCards` talon needs to be passed a gift card code.

Is this the easiest way to do that?